src/helpers/index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable require-jsdoc */
```

Comment:
@william000000 I think we have a validation folder? This can go inside the validation folder rather than in the helpers folder

Reply:
Feedback Implemented